tests/test_finance.py

```diff
@@ -546,8 +547,16 @@ def test_market_minute_window(self):
         self.assertTrue(all(friday == minutes[31:421]))
         self.assertTrue(all(thursday == minutes[421:]))
 
+    def test_min_date(self):
+        min_date = pd.Timestamp('2016-03-04', tz='UTC')
+        env = TradingEnvironment(min_date=min_date)
+
+        self.assertLessEqual(env.first_trading_day, min_date)
```

Comment:
If we're specifying a minimum, shouldn't the `first_trading_day` be greater than or equal to it?